fe/fe-core/src/main/java/com/starrocks/common/util/ProfileManager.java

```diff
@@ -182,16 +179,9 @@ public String pushProfile(ProfilingExecPlan plan, RuntimeProfile profile) {
 
         writeLock.lock();
         try {
-            if (queryType != null && queryType.equals("Load")) {
```

Comment:
It is better to maintain an independent profile map to separately limit the number, as the logic for loading profiles and querying profiles is quite different. Loading profiles may involve batch loads scheduled for execution at midnight, and these profiles are expected to be retained for a longer period without being evicted by query profiles.

Reply:
I don't think it's necessary, we won't turn on profiles all the time in an online environment. we'll only turn on reporting for specific queries and large query imports, otherwise any profile will be meaningless.